report-pox-krypton.js

```diff
@@ -310,8 +310,31 @@ const staging_db_path = 'chainstate/chain-00000080-testnet/blocks/staging.db'
 //                                            vtxindex INT NOT NULL
 //     );
 
-const data_root_path = `${root}${process.argv[3] || process.argv[2]}`
-const use_txs = process.argv[2] === '-t'
+// const data_root_path = `${root}${process.argv[3] || process.argv[2]}`
+// const use_txs = process.argv[2] === '-t'
+
+let use_txs = false;
+let use_csv = false;
+var myArgs = process.argv.slice(2);
```

Comment:
use `const` instead

Reply:
and variable use `snake_case` while function names use `camelCase`

Reply:
Ignore 86908d2, fixed in acfece6, no coffee yet :coffee: